.circleci/config.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       - run:
           name: Connect new -> old
           command: sudo ../root_version/build/etserver --daemon; sudo cp ../root_version/build/etterminal /usr/bin/etterminal; sleep 1; build/et -c "ls" localhost --logtostdout --verbose=9
+      - run:
+          name: Connect new -> old (ipv6)
+          command: sudo ../root_version/build/etserver --daemon; sudo cp ../root_version/build/etterminal /usr/bin/etterminal; sleep 1; build/et -c "ls" 0.0.0.0.0.0.0.1 --logtostdout --verbose=9
```

Comment:
Shouldn't this be 0:0:0:0:0:0:0:1 (with colons instead of dots)?

Reply:
Yep, just pushed fix! I also removed the starting of etserver and cp of etterminal since that is already done.

Reply:
Looks like maybe the ssh daemon is not configured to work with ipv6: 

```
ssh: connect to host ::1 port 22: Connection refused
```